docs/content/doc/advanced/config-cheat-sheet.en-us.md

```diff
@@ -140,6 +140,7 @@ Values containing `#` or `;` must be quoted using `` ` `` or `"""`.
 - `CERT_FILE`: **custom/https/cert.pem**: Cert file path used for HTTPS.
 - `KEY_FILE`: **custom/https/key.pem**: Key file path used for HTTPS.
 - `STATIC_ROOT_PATH`: **./**: Upper level of template and static files path.
+- `STATIC_CACHE_TIME`: 6h: Static resouces, includes resouces on custom/, public/ and all uploaded avatars web browser cache time.
```

Comment:
`resouces` -> `resources`

Reply:
Also, maybe make that `6h: ` to `Default: 6h.` at the end of the description.

Reply:
We usually have the default in bold.

How about:

```suggestion
- `STATIC_CACHE_TIME`: **6h**: Web browser cache time for static resources on `custom/`, `public/` and all uploaded avatars.
```